packages/schematics/src/collection/ngrx/schema.d.ts

```diff
@@ -1,4 +1,4 @@
-export interface Schema {
+export interface NgrxOptions {
```

Comment:
Could you change it back? We use the same convention in all our schematics. The Schema type represents the contents of schema.json.

Reply:
fixed.